dev/com.ibm.ws.openapi.ui/update-openapi-ui.sh

```diff
@@ -0,0 +1,77 @@
+#*******************************************************************************
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+#!/usr/bin/env bash
+
+# Check git username was provided as an argument
+if ! [ $# -eq 1 ]; then
+    echo "Missing git username argument."
+    exit 2
+else
+    git=$1
+fi
+
+# Ensure that a Node 16 environment is used
+if ! node -v | grep -q "^v16"; then
```

Comment:
We should see if we can use Node 18 and 20. 16 is now EoL so saying we must be on 16 is restrictive unless we find the build process doesn't work with newer Node versions. So I would try with 18 and 20 to see if the generated result of the npm build process is different. if it isn't then we should change the check to be >16 and divisible by 2 as we want it built with an LTS version